src/plugins/unified_histogram/public/chart/utils/get_lens_attributes.ts

```diff
@@ -205,6 +205,13 @@ export const getLensAttributes = ({
       filters,
       query,
       visualization,
+      ...(dataView &&
+        dataView.id &&
+        !dataView.isPersisted() && {
+          adHocDataViews: {
+            [dataView.id]: dataView.toSpec(),
```

Comment:
```suggestion
            [dataView.id]: dataView.toSpec(false),
```
to make `panelJSON` smaller inside dashboard SO